src/js/control-bar/play-toggle.js

```diff
@@ -58,6 +59,29 @@ class PlayToggle extends Button {
   }
 
   /**
+   * This gets called whenever the player seeks. We want to change the replay button
+   * back to a play/pause button on any seek because you cannot seek to the very end
+   * of a video
+   *
+   * @param {EventTarget~Event} [event]
+   *        The event that caused this function to run.
+   *
+   * @listens Player#seeked
+   */
+  handleSeeked(event) {
+    // check to see if the video has ended yet
+    if (!this.hasClass('vjs-ended')) {
```

Comment:
oh, we probably should use `this.ended()` instead of checking the class.

Reply:
Yeah I guess that will work, should be `this.player_.ended()` right?

Reply:
oops, yes, it's on the player